pants.toml

```diff
@@ -1,5 +1,14 @@
 [GLOBAL]
 pants_version = "2.13.0rc2"
+backend_packages = [
+  # python
+  "pants.backend.python",
+  "pants.backend.experimental.python",
```

Comment:
OK. I left a comment. This enables support for the python publish goal (`./pants publish`) and the twine subsystem. The publish goal uses twine to publish wheels to pypi.

relevant docs are not linked in the sidebar because they are in the experimental backend.
https://www.pantsbuild.org/docs/python-publish-goal
https://www.pantsbuild.org/docs/reference-twine